Eastern-Research-Group/mywaterway
# Bugfix/441 cyan overlapping dates

## Related Issues:
* [HMW-441](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-441)

## Main Changes:
* Updated CyAN date range calculations to account for Daylight Savings Time. This involves checking if there is a difference in the time zone offsets of two dates.

## Steps To Test:
1. CyAN's CORS settings do not allow access from `localhost`. I tested with mock data, but it's still necessary to check that the correct calculation is used in the initial service request. I attempted to switch to the `proxyFetch` function, but didn't have any luck, so may need to double-check after merging into develop.
